### ISSUE TITLE
Allow additional info in serialization

### ISSUE THREAD

**Issue by enikao** [LionWeb-io/specification#67]
We should allow to pass additional info in serialization format at some well-defined place.

Example:
```json
{
    "serializationFormatVersion": "1",
    "additionalBla": "Blubb", // additional info here
    "nodes": [
      {
        "type": "Library",
        "id": "idBobLib",
        "properties": {
          "LibraryName": "Bob's library"
        },
        "children": {
          "LibraryBooks": [
            "idExplorer"
          ]
        },
        "parent": null
      }
  ]
}
```

**Comment by enikao** [comment]
**Decision on 2023-10-13: Don't allow additional fields (for now)**

Rationale:
* Additional info can be outside the LW serialization structure (see comment above)
* Even if a LW parser would ignore additional fields, it would not keep them around / send them back (or, if we required that, would pose a lot of additional effort on LW implementations).
* If we allowed additional fields in the future, we could still be backwards-compatible.

We might revise this decision in the future if we find a good use case.
